.circleci/config.yml

```diff
@@ -5,16 +5,28 @@ jobs:
 
     docker:
         - image: gcc:5.4.0 
-
+    
     steps:
 
       - checkout
 
       - run:
           name: update common
-          command: git submodule update
+          command: git submodule update --init
```

Comment:
In the [documentation](https://circleci.com/docs/2.0/configuration-reference/#checkout) it's recommended to use:
```
- run: git submodule sync
- run: git submodule update --init
```
But I'm not sure if the `sync` is really needed.